subscriptions/reporting/subscription_reports.rst

```diff
@@ -0,0 +1,129 @@
+  unlikely to occur in the ordinary course of businesses.
+
+- **Customer Retention**: This concept consists of engaging existing customers to continue buying
+  products or services from your business. As you may know, customer retention is quite a challenge
+  because you must prove yourselves worthy of your customers' trust.
+
+- **Churn Rate**: Also known as the rate of attrition or customer churn, the churn rate can be
+  defined, in this case, as the percentage of service subscribers who discontinue their subscriptions
+  within a given time period. We can distinguish two types of Churn:
+
+  - **Logo Churn**: It corresponds to the subscription cancellation rate.
+
+  - **Revenue Churn**: It corresponds to the monthly recurring revenue loss rate.
+
+  .. note::
+     Let's imagine an increase of 2$ on a service subscription.
+
+     - We loss three customers out of the initial 20 which gives us a **Logo Churn** of 15%.
+     - | Therefore, the 56$ of MRR difference out of the initial 600$ gives us a **Revenue Churn**
+       | of 9,33%.
+
+     .. image:: media/difference-between-logo-churn-and-revenue-churn.png
+        :align: center
+        :alt: Difference between logo churn and revenue churn in Odoo Subscriptions
+
+     As a reminder, though they seem to evolve in the same direction most of the time, it might not
+     be the case all time.
+
+- **Customer Lifetime Value (CLV)**: Very important measure which indicates how much revenue can be
+  expected for a customer during his/her entire contract. This approach emphasizes the importance of
+  customer retention, shifting our focus from a quarterly or yearly approach to a long-term vision.
+
+Check out the different kinds of reports you can access from the **Odoo Subscriptions** application.
+
+Subscriptions analysis report
+=============================
+
+Go to :menuselection:`Subscriptions --> Reporting --> Subscriptions`. From there, you can change the
+*Measures*. By default, Odoo uses the *Monthly Recurring Revenue*. In addition to that, you can
+choose *Quantity*, *Recurring Price*, *Untaxed Recurring Price*, *Yearly Recurring Revenue* and
+*Count*. For this example, *Quantity* is added. This way, you can review both of these measures at
+the same time. You can even *Group By Start Date* and more precisely, by *Week* to get a clear view
+of your report.
+
+.. image:: media/subscriptions-analysis-report.png
+  :align: center
+  :alt: Subscriptions analysis report in Odoo Subscriptions
+
+To ease the analysis of your data, you can click on the *Insert In Spreadsheet* button. The interface
+of the Odoo spreadsheet feature has been improved in our most elaborated version of Odoo. Everything
+has been done to facilitate the user experience.
+
+.. image:: media/subscriptions-analysis-report-and-spreadsheet-feature.png
+  :align: center
+  :alt: Subscriptions analysis report using the spreadsheet feature in Odoo Subscriptions
```

Comment:
So, I believe this is to be included on the update of the doc for 14, just showing here what V13 can do (:

Reply:
Waiting too :)

Reply:
And done :)